layout/_scripts/mermaid.swig

```diff
@@ -0,0 +1,12 @@
+{% if theme.mermaid.enable %}
+  <script src="{{ url_for(theme.js) }}/src/mermaid.js?v={{ version }}"></script>
```

Comment:
How about `theme.vendors.mermaid` option?
```yml
vebdors:
  mermaid: //cdn.jsdelivr.net/npm/mermaid@8.0.0/dist/mermaid.min.js
```
 And creat a repo named theme-next-mermaid. Just like before, `_third-party/mermaid.swig` contains all code.

Reply:
Yep, this is variant too.

Reply:
@ivan-nginx @1v9 Waiting for answers from https://github.com/theme-next/hexo-theme-next/issues/655. It's not easy to maintain all these `theme-next-*` repos, to keep them up to date

Reply:
@stevenjoezhang I mean put core code in master branch, mermaid.min.js can use CDN and main org repo, no need to put large js file in master branch, is it possible?

Reply:
@1v9 or we only use CDN? Like mathjax.

Reply:
@ That's OK, I have not created quicklink repo yet 😫. I used jsdeliver mermaid.min.js on my site, it's really fast!

Reply:
OK, I'll update. 😉